pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -1748,7 +1748,7 @@ func TestValidatePipelineParameterVariables_Failure(t *testing.T) {
 		}},
 		expectedError: apis.FieldError{
 			Message: `params names must be unique, the same param: duplicate-param is defined multiple times at`,
-			Paths:   []string{"[0].params[1].name", "[0].params[2].name"},
+			Paths:   []string{"[0].params.params[1].name, [0].params.params[2].name"},
```

Comment:
this path doesn't look right 🤔

Reply:
yup, you are right, dropped the extra `params` 🙃